src/web/components/SharingPermission/SharingPermissionsTable.tsx

```diff
@@ -49,7 +49,9 @@ function DeletePermissionDialog({
   };
 
   const showDeletionNotice = (participant: SharingParticipant) => {
-    const remainSources = participant.addedBy.filter((source) => source !== 'Manually Added');
+    const remainSources = (
+      participant.addedBy.filter((source) => source !== 'Manual') as ParticipantTypeDTO[]
```

Comment:
nit: is there a reason we still keep the string here rather than constant? is that breaking type check?

Reply:
Because we've got type checking now it doesn't really matter - if you get the string wrong it's a type error. I've updated it anyway 🙂